types/yeoman-generator/index.d.ts

```diff
@@ -872,6 +892,29 @@ declare class Generator<T extends Generator.GeneratorOptions = Generator.Generat
      */
     yarnInstall(pkgs?: string | string[], options?: object, spawnOptions?: SpawnOptions): void;
 
+    // actions/package.json
+    /**
+     * Adds dependencies to the destination `package.json`.
+     *
+     * @param dependencies
+     * The packages to add.
+     *
+     * @returns
+     * The newly added dependencies.
+     */
+    addDependencies(dependencies: { [pkg: string]: string } | string | string[]): Promise<{ [pkg: string]: string }>;
```

Comment:
just one tiny suggestion, why not use `Record<string,string>` instead of `{ [pkg: string]: string }`?

Reply:
Alright!
I replaced all objects with `Record`s now

Thanks für the advice 😄